Source/Microsoft.Teams.Apps.FAQPlusPlus/Cards/CardHelper.cs

```diff
@@ -87,15 +87,33 @@ public static string GetTicketDisplayStatusForSme(TicketEntity ticket)
         }
 
         /// <summary>
-        /// Returns a string that will display the given date and time in the user's local time zone, when placed in an adaptive card.
+        /// Returns a string that will display the given date and time in the user's local time zone in a thumbnail card.
         /// </summary>
         /// <param name="dateTime">The date and time to format.</param>
         /// <param name="userLocalTime">The sender's local time, as determined by the local timestamp of the activity.</param>
         /// <returns>A datetime string.</returns>
         public static string GetFormattedDateInUserTimeZone(DateTime dateTime, DateTimeOffset? userLocalTime)
         {
-            // Adaptive card on mobile has a bug where it does not support DATE and TIME, so for now we convert the date and time manually.
-            return dateTime.Add(userLocalTime?.Offset ?? TimeSpan.FromMinutes(0)).ToShortDateString();
+            // Keeping date format for ar-sa as invariant, since we don't want to convert the dates to islamic calendar dates.
+            if (CultureInfo.CurrentCulture.Name.Equals("ar", StringComparison.OrdinalIgnoreCase) || CultureInfo.CurrentCulture.Name.Equals("ar-SA", StringComparison.OrdinalIgnoreCase))
+            {
+                return dateTime.Add(userLocalTime?.Offset ?? TimeSpan.FromMinutes(0)).ToString("dd/MM/yy", CultureInfo.InvariantCulture);
```

Comment:
Take this from CurrentCulture.DateTimeFormat.ShortDatePattern

Reply:
Done. Please have a look. Thanks